src/constants/tags.hpp

```diff
@@ -8,14 +8,14 @@
 
 CONSTANT_TYPE( avogadro );
 CONSTANT_TYPE( boltzmann );
-CONSTANT_TYPE( G );
+CONSTANT_TYPE( k );
 CONSTANT_TYPE( elementaryCharge );
+CONSTANT_TYPE( G );
 CONSTANT_TYPE( newtonianGravitation );
-CONSTANT_TYPE( k );
 CONSTANT_TYPE( planck );
 CONSTANT_TYPE( h );
-CONSTANT_TYPE( hbar );
-CONSTANT_TYPE( reducedPlanck );
+// CONSTANT_TYPE( hbar );
+// CONSTANT_TYPE( reducedPlanck );
 CONSTANT_TYPE( speedOfLight );
 CONSTANT_TYPE( c );
```

Comment:
`c` and `speedOfLight` ought not be instances of different types. likewise with boltzmann/k and newtonianGraviation/G (is there another gravitation constant?)

Reply:
```
#define ALIAS(original, alias) constexpr auto alias = original;
ALIAS(boltzman, k)
...
#undef ALIAS 
```

Reply:
I like this idea, but it doesn't quite work.

```cpp
#define ALIAS( original, alias )  static constexpr auto alias = original;
ALIAS( boltzmann, k )
#undef ALIAS
```

gives the following compile error:

```
      static_assert failed "hana::at_key(map, key) requires the 'key' to be present in the 'map'"
            static_assert(!decltype(hana::is_nothing(MaybeIndex{}))::value,
            ^             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
It seems that using the `ALIAS` macro doesn't put things into the map as desired.